packages/frontend/core/src/features/auth/login/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const LOGIN_WITH = "username";
+export const LOGIN_NAME = "login";
+export const SIGNUP_NAME = "create account";
+
+export const toTitleCase = (str: string): string => {
```

Comment:
are we talking about `camelCase` here rather than `Title Case Which Is Like This`?

Reply:
The intention of the function is only to capitalize the first letter, e.g. create account -> Create account. Agree the function name isn't that great. How about `capitalize`?